editor/header/tools/preview-button.js

```diff
@@ -16,10 +15,14 @@ import { getEditedPostPreviewLink } from '../../selectors';
 
 function PreviewButton( { link } ) {
 	return (
-		<Button href={ link } target="_blank">
-			<Dashicon icon="visibility" />
+		<IconButton
+			href={ !! link ? link : undefined }
```

Comment:
Was there a case where `link` was not `undefined` but not intended to be used as the `href` to prompt this change?